website/docs/r/license.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_license"
+sidebar_current: "docs-vsphere-resource-license"
+description: |-
+  Provides a VMware vSphere license resource. This can be used to add and remove license keys.
+---
+
+# vsphere\_license
+
+Provides a VMware vSphere license resource. This can be used to add and remove license keys.
+
+## Example Usage
+
+```hcl
+resource "vsphere_license" "licenseKey" {
+  license_key = "452CQ-2EK54-K8742-00000-00000"
+
+  labels {
+    VpxClientLicenseLabel = "Hello World"
+    Workflow = "Hello World"
+  }
+  
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `license_key` - (Required) The license key to add.
+* `labels` - (Optional) A map of key/value pairs to be attached as labels (tags) to the license key.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `edition_key` - The product edition of the license key.
+* `total` - Total number of units (example: vCPUs) contained in the license.
```

Comment:
Sorry I missed this before @girishramnani but this needs to be changed to CPUs as well - I don't think I've seen a license have vCPUs at all.